snap7/__init__.py

```diff
@@ -3,14 +3,13 @@
 """
 import pkg_resources
 
-import snap7.server as server
 import snap7.client as client
```

Comment:
I think there is the async client missing here.

Reply:
it should first be a bit more mature